exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/_metric_exporter/__init__.py

```diff
@@ -168,8 +168,8 @@ def _translate_data(
             )
         )
 
-    def export(self, metrics: Sequence[Metric]) -> MetricExportResult:
+    def export(self, metrics: Sequence[Metric], *args, **kwargs) -> MetricExportResult:
```

Comment:
Do we need `*args`? I would suggest just have kwargs.

Reply:
All right, removed `*args`.